niri-config/tests/wiki-parses.rs

```diff
@@ -0,0 +1,99 @@
+use include_dir::{include_dir, Dir};
+
+static WIKI_DIR: Dir = include_dir!("$CARGO_MANIFEST_DIR/../wiki");
```

Comment:
I think instead of this it's better to just normally walk the folder in the test. There's already some heavier files there like png images, this could blow up unexpectedly.

Reply:
The issue is that we don't know where the wiki directory could be at runtime IIRC. E.g. I run `cargo test` inside of the workspace vs. inside of `niri-config`

Reply:
I think it is always consistent? Also you still have CARGO_MANIFEST_DIR, you can embed that at compile time

Reply:
used env! instead. works as well indeed